recipes/searvey/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "searvey" %}
+{% set version = "0.1" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oceanmodeling/searvey/archive/v{{ version }}.tar.gz
+  sha256: 293bae61773fd95b373df2909540cb3ce30f97526eab6c0694181dcd4b53464b
+
+build:
+  noarch: python
+  script_env:
+    - __version__={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
+  number: {{ build }}
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+    - poetry-core >=1.0.0
```

Comment:
I'm not a poetry expert but you may need to add `poetry-dynamic-versioning` b/c that is declared in the pyproject.toml upstream.

Reply:
`poetry-dynamic-versioning` loads successfully but then fails with:

```
   File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/poetry/core/masonry/api.py", line 43, in prepare_metadata_for_build_wheel
      poetry = Factory().create_poetry(Path(".").resolve(), with_dev=False)
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/poetry_dynamic_versioning/patch.py", line 52, in alt_poetry_create
      name = _get_and_apply_version(
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/poetry_dynamic_versioning/__init__.py", line 317, in _get_and_apply_version
      version = _get_version(config)
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/poetry_dynamic_versioning/__init__.py", line 172, in _get_version
      version = Version.from_vcs(
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/dunamai/__init__.py", line 1322, in from_vcs
      return cls._do_vcs_callback(vcs, pattern, latest_tag, tag_dir, tag_branch, full_commit)
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/dunamai/__init__.py", line 1352, in _do_vcs_callback
      return callback(**kwargs)
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/dunamai/__init__.py", line 1290, in from_any_vcs
      vcs = _detect_vcs()
    File "/private/tmp/pip-build-env-3c6cj474/overlay/lib/python3.10/site-packages/dunamai/__init__.py", line 277, in _detect_vcs
      raise RuntimeError("Unable to detect version control system.")
  RuntimeError: Unable to detect version control system.
  error: subprocess-exited-with-error
```

Reply:
Maybe poetry has a "_version.py" file functionality like setuptoos-scm but I don't know enough about it to be sure. Let's leave the env variable for now.